tests/plugins/test_orchestrate_build.py

```diff
@@ -97,8 +107,10 @@ def mock_workflow(tmpdir):
     df = df_parser(df_path)
     setattr(workflow.builder, 'df_path', df.dockerfile_path)
 
-    build = '{"spec": {"strategy": {"customStrategy": {"from": \
-            {"name": "some_image:latest", "kind": "DockerImage"}}}}}'
+    build = '{"spec": {"strategy": {"customStrategy": {"from":' + \
+            '{"name": "registry/some_image@sha256:123456", "kind": "DockerImage"}}}},' + \
+            '"metadata": {"annotations": {"from": "{\\"name\\": \\"registry' + \
+            '/some_image:latest\\", \\"kind\\": \\"DockerImage\\"}"}}}'
```

Comment:
There will come a point where it might be easier to just use `json.dumps()` on a dict to get these strings.

Reply:
I think we've past that point.